src/renderer/shared/ui-entities/AssetIcon/AssetIcon.stories.tsx

```diff
@@ -35,18 +35,23 @@ export const Monochrome: Story = {
   },
 };
 
-const allPossibleAssets = uniqBy(
-  (chains as Chain[]).flatMap(({ assets }) => assets),
-  ({ symbol }) => symbol,
-);
-
 export const VariantsDerivedFromConfig: Story = {
   args: {
     style: 'colored',
     size: 32,
   },
   render: args => {
     const [query, setQuery] = useState('');
+    const [chains, setChains] = useState<Chain[]>([]);
+
+    useEffect(() => {
```

Comment:
You can use loaders
https://storybook.js.org/docs/writing-stories/loaders#fetching-api-data